ui/app/routes/vault/cluster/clients/index.js

```diff
@@ -53,13 +53,20 @@ export default Route.extend(ClusterRoute, {
     let activity = await this.getActivity(license.startTime); // returns client counts using license start_time.
     let monthly = await this.getMonthly(); // returns the partial month endpoint
 
+    // all times to the component must be mm,yyyy ex: "1,2021"
+    let endTimeFromLicense = `${activity.endTime.split('-')[1].replace(/^0+/, '')},${
+      activity.endTime.split('-')[0]
+    }`;
+    let startTimeFromLicense = `${license.startTime.split('-')[1].replace(/^0+/, '')},${
```

Comment:
This line and the one above are very similar, it might be worth splitting into another method. Then It might be easier to define what you expect from the license and what you expect to get back, and that makes it easier to test too!

Reply:
We refactored this, should look a little more DRY now.